src/cdk/cdk.context.json

```diff
@@ -15,5 +15,11 @@
       "PrefixListId": "pl-3b927c52",
       "Identifier": "pl-3b927c52"
     }
+  ],
+  "cc-api-provider:account=431151642132:expectedMatchCount=exactly-one:propertiesToReturn.0=PrefixListId:propertyMatch.PrefixListName=com.amazonaws.global.cloudfront.origin-facing:region=us-east-2:typeName=AWS$:$:EC2$:$:PrefixList": [
```

Comment:
We shouldn't commit this as it's account specific. This whole file IMO should be removed from the repo as it's generated on every synth.

Reply:
yep agreed. We should probably add to .gitignore but can be done on a different PR.